Northstar.Custom/mod/scripts/vscripts/table_utilities.nut

```diff
@@ -0,0 +1,80 @@
+untyped
+globalize_all_functions
+
+string function GetTableString( table t, string index, string defaultVal = "" )
+{
+    try
+    {
+    return expect string(t[index])
+    }
+    catch (ex)
+    {
+
+    }
+    return defaultVal
+}
```

Comment:
It seems kinda bad to me to be using `try catch` for all of these functions? I'd be more happy with using something like `if (index in t)` but untyped might ruin that one